src/image.tsx

```diff
@@ -53,6 +53,9 @@ const exportableLoader: ImageLoader = ({ src: _src, width, quality }) => {
   const outputDir = `/${
     config.imageDir ? config.imageDir.replace(/^\//, '').replace(/\/$/, '') : '_next/static/chunks/images'
   }`
+  const externalOutputDir = `/${
+    config.externalImageDir ? config.externalImageDir.replace(/^\//, '').replace(/\/$/, '') : '_next/static/media'
+  }`
```

Comment:
Looks like the leading slash is unnecessary!

Reply:
Apologies for taking a while to get back to this, was on holiday.